pinot-controller/src/main/java/org/apache/pinot/controller/util/ConsumingSegmentInfoReader.java

```diff
@@ -132,7 +133,8 @@ private Map<String, List<SegmentConsumerInfo>> getConsumingSegmentsInfoFromServe
     if (failedParses != 0) {
       LOGGER.warn("Failed to parse {} / {} segment size info responses from servers.", failedParses, serverUrls.size());
     }
-    return serverToConsumingSegmentInfoList;
+    return new ConsumingSegmentsInfoFromServersResponse(
+        serverToConsumingSegmentInfoList, serviceResponse._failedResponseCount);
```

Comment:
Since we are gathering the servers not responded, should we also include un-parsable responses as well?